ntbs-service/Pages/Notifications/LegacyView.cshtml.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using ntbs_service.DataMigration;
+using ntbs_service.Models;
+using ntbs_service.Services;
+
+namespace ntbs_service.Pages.Notifications
+{
+    public class LegacyView : PageModel
+    {
+        private readonly ILegacySearchService _legacySearchService;
+        private readonly INotificationImportService _notificationImportService;
+
+        [BindProperty] 
+        public NotificationBannerModel NotificationBanner { get; set; }
+        
+        [BindProperty(SupportsGet = true)]
+        public string LegacyNotificationId { get; set; }
+        
+        public string RequestId { get; set; }
+        public ImportResult LegacyImportResult { get; set; }
+
+        public LegacyView(ILegacySearchService legacySearchService, INotificationImportService notificationImportService)
+        {
+            _legacySearchService = legacySearchService;
+            _notificationImportService = notificationImportService;
+        }
+        
+        public async Task<IActionResult> OnGetAsync()
+        {
+            await GetLegacyNotificationDetailsForBanner();
```

Comment:
Currently there is no NotFound shown if you navigate to this page with a non existent legacy id - I think we should check for the notification existing and redirect to a NotFound if it doesn't exist